frontend/public/components/pdb/PDBFormPage.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { PageHeading } from '@console/internal/components/utils';
+import { SyncedEditor } from '@console/shared/src/components/synced-editor';
+import { EditorType } from '@console/shared/src/components/synced-editor/editor-toggle';
+import { CreateYAML } from '@console/internal/components/create-yaml';
+import {
+  K8sResourceKind,
+  PodDisruptionBudgetKind,
+  LabelSelector,
+} from '@console/internal/module/k8s';
+import { PodDisruptionBudgetModel } from '../../models';
+import PDBForm from './PDBForm';
+import { pdbToK8sResource } from './pdb-models';
+import { useK8sWatchResource } from '../utils/k8s-watch-hook';
+
+const LAST_VIEWED_EDITOR_TYPE_USERSETTING_KEY = 'console.PDBForm.editor.lastView';
+
+export const PDBFormPage: React.FC<{
+  location: { search: string };
+  match: Match;
+}> = (props) => {
+  const { t } = useTranslation();
+  const match = {
+    params: { ...props.match.params, plural: PodDisruptionBudgetModel.plural, appName: '' },
+    isExact: true,
+    url: '',
+    path: '',
+  };
+  const { location } = props;
+  const searchParams = new URLSearchParams(location.search);
+  const kind = searchParams.get('kind');
+  const name = searchParams.get('name');
+
+  const [resource] = useK8sWatchResource<K8sResourceKind>({
+    kind,
+    name,
+    namespaced: true,
+    namespace: match.params.ns,
+  });
+
+  const [pdbResources] = useK8sWatchResource<PodDisruptionBudgetKind[]>({
+    kind: PodDisruptionBudgetModel.label,
```

Comment:
`kind` and `label` are two different things. We should be using the full group/version/kind.